docs/guides/machines.md

```diff
@@ -123,7 +126,7 @@ const testLightMachine = lightMachine.withContext({
 ```
 
 ::: warning
-This will _not_ do a shallow merge of the original `context`, and will instead _replace_ the original `context` with the `context` provided to `.withContext(...)`. You can still "merge" contexts manually, by referencing `machine.context`:
+This will _not_ do a shallow merge of the original `context`, and will instead _replace_ the original `context` with the `context` provided to `.withContext(...)`. You can still “merge” contexts manually, by referencing `machine.context`:
 
 ```js
 const testLightMachine = lightMachine.withContext({
```

Comment:
Something funky going on with GitHub here, so I’ll put all the comments for the lines further down on this line:

91: Is there a way to describe which object to make it clear which object we mean?

129: Is “this” extending the machine with a different initial context?